### DISCUSSION TITLE
Announcement pattern

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3155] [Office hours] [answered]
### Background

The Reverse ETL team is moving a feature from one page to another and I'm curious to know if we there's a pattern / component we use for these changes.

### Feedback

We’re exploring using the alert component as we want it to be dismissible, but I'm not sure what the threshold for “high-priority or time-sensitive” is for us to use it?

Evergreen has this tour component that great at highlighting pieces of the UI, making the information more contextual, but i don’t think paste has an equivalent.

[Figma file](https://www.figma.com/file/VHqhSLhLrjbZplLzyzt6ab/%5BDesign%5D-Reverse-ETL-post-GA?node-id=148%3A323968&t=ilCUBDxkOdrg9mkK-1)

### Stage of design

Exploration

### Are you looking for visual design support?

- [X] Yes

**Comment by serifluous** [comment, marked as answer]
Hey @char-mainewang, you have a few options here!

1. One way to consider whether it's high-priority or time-sensitive is to think about what the risk is if a user doesn't see this information. Are they going to be blocked in some critical workflow if they miss it? If the risk isn't super obvious, you could use a Callout: https://paste.twilio.design/components/callout

The visual design of Paste Callouts and Alerts are pretty close now, so the main difference comes down to how screenreaders announce the text in the Alerts vs. Callouts. You can read more about that here: https://paste.twilio.design/components/alert#accessibility

2. If you feel like the Evergreen tour component is a better solution, you could use [Popover](https://paste.twilio.design/components/popover#using-state-hooks) and have it trigger on page load, though you'll need to add a button or something similar to the page to function as a trigger. Can you share a demo or explain how the Evergreen component works? I don't see it on the Evergreen doc site, but it might be something that's composable with [Box](https://paste.twilio.design/primitives/box) and Popover.

**Reply by char-mainewang** [reply]
We chatted about using the interactive badge + popover during office hour, i'm exploring putting the badge on the tab where users can configure that setting, would this introduce any accessibility issues?

<img width="864" alt="image" src="https://user-images.githubusercontent.com/87334135/231884722-5dde8014-7178-434e-bb18-7f2722418332.png">

**Reply by SiTaggart** [reply]
Hi

Unfortunately, yes, you can't place interactive items inside another interactive element like a tab, as there is no way to navigate to them and sometimes no way to detect they are there.

Sorry

**Comment by libertycarlin** [comment]
Thank you for joining us at Office Hours @char-mainewang! Here is the[ link](https://www.loom.com/share/9a301cc5dd98450c89465b900c455ec2) to the recording and password is in the help-design-system channel
